packages/components/checkbox/src/checkbox-button.vue

```diff
@@ -8,8 +8,8 @@
       :name="name"
       :tabindex="tabindex"
       :disabled="isDisabled"
-      :true-value="trueValue || trueLabel"
-      :false-value="falseValue || falseLabel"
+      :true-value="trueValue ?? trueLabel"
+      :false-value="falseValue ?? falseLabel"
       @change="handleChange"
```

Comment:
Do you think the logic operation on line 4 needs to be adjusted? 🤔

Reply:
I don't think it needs to be adjusted, `trueValue` should be a truthy value.

Reply:
Ok.